projects/fsastorefrontlib/src/cms-components/my-account/inbox/inbox-tab/inbox-messages/inbox-messages.component.html

```diff
@@ -71,20 +72,27 @@
         <span class="checkbox-text fs-icon icon-ok"></span>
       </label>
     </div>
-    <div
-      class="col-8 text-center"
-      (click)="displayMobileGroups = !displayMobileGroups"
+    <ng-container
+      *ngIf="{
+        subject: defaultSortOrder,
+        content: defaultSortOrder,
+        sentDate: defaultSortOrder
+      } as sorts"
```

Comment:
If we are **not** going to have sorting on mobile, this can be removed. Even if we did, maybe it would be optimal to move it to the top, to wrap all heading elements inside it. That way we could avoid 2 of the same code blocks.